frontend/public/components/machine-config-pool.tsx

```diff
@@ -0,0 +1,261 @@
+/* eslint-disable no-undef, no-unused-vars */
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+import { MachineConfigPoolModel } from '../models';
+import { referenceForModel, MachineConfigPoolKind } from '../module/k8s';
+import { machineConfigReference, MachineConfigPage } from './machine-config';
+import { Tooltip } from './utils/tooltip';
+import { Conditions } from './conditions';
+import { errorModal } from './modals';
+import { machineReference, MachinePage } from './machine';
+import {
+  ColHead,
+  DetailsPage,
+  List,
+  ListHeader,
+  ListPage,
+} from './factory';
+import {
+  Kebab,
+  navFactory,
+  pluralize,
+  ResourceKebab,
+  ResourceLink,
+  resourcePath,
+  ResourceSummary,
+  SectionHeading,
+  Selector,
+  togglePaused,
+  WorkloadPausedAlert,
+} from './utils';
+
+export const pauseAction = (kind, obj) => ({
```

Comment:
It looks like a lot of things are being exported in this module that don't need to be.